### DISCUSSION TITLE
Rich preview of a stream

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5333] [Ideas]
To choose what to watch, some users focus on categories. Now, to find out the categories of all channels, you need to check them one by one. I suggest simplifying this task and increasing the usefulness of the preview. If you have concerns that this may interfere, then it is enough to make the feature optional.

![Screenshot 2024-04-18 at 15 21 29](https://github.com/Chatterino/chatterino2/assets/60105060/ecb6c5fc-e7e0-4400-892d-032ba6def52c)

**Comment by Mm2PL** [comment]
I don't think including "other live streams" in a tooltip of a single split is a good idea.
A propos formatting: red on black might be hard to read. Same goes for the dark-gray but it is slightly better. The title going after the preview is okay. I like the that you've made the title bold, makes it stand out as a separate element more. We could also probably play with font sizing here too, making the title one or two sizes bigger.

**Reply by x9136** [reply]
The visual part can be any, as well as the solution, it is not as important as the ability to see all categories at once.

As for the location, I don't see a better place that will allow you not to complicate Chatterino and exclude the addition of unnecessary actions. I don't see any point in complicating it if the simple option will work well enough.

**Reply by Mm2PL** [reply]
Maybe this would be a fit to `/live`? Instead of `{name} is live!`, the messages could be `{name} is live in {category}: {title}` or something similar.
